internal/provider/data_logging_endpoints_test.go

```diff
@@ -0,0 +1,28 @@
+package provider
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccLoggingEndpointsDataSource(t *testing.T) {
```

Comment:
I think we should remove the Github test which tests this on Prod. It's risky having this test on Prod as if it fails we'd have leaked data. Please could you remove the Github workflow to run on Prod?